src/core.js

```diff
@@ -608,7 +614,11 @@ $.extend( $.validator, {
 				return element.validity.badInput ? false : $element.val();
 			}
 
-			val = $element.val();
+			if ( element.hasAttribute( "contenteditable" ) ) {
```

Comment:
I believe hasAttribute isn't supported completely by IE8 either.
This may be better written as: $(this).is( "[contenteditible]" )